sys/net/include/rpl.h

```diff
@@ -8,7 +8,11 @@
  */
 
 /**
+ * @defgroup    rpl RPL Header
```

Comment:
According to #2422 this should be `net_rpl`. I also would remove the "Header". The other protocol implementations don't use it either: http://riot-os.org/api/group__net.html